### DISCUSSION TITLE
Can't make internal ghcr images public

### DISCUSSION THREAD

**Discussion by aptalca** [community/community#27164] [Actions]
<p>Hi there,</p>
<p>We as an org at <a href="https://github.com/linuxserver" rel="noopener nofollow ugc">linuxserver.io</a> have been publishing over a hundred images on <a href="http://ghcr.io" rel="noopener nofollow ugc">ghcr.io</a> and it had been working fine until now.</p>
<p>Our images are pushed by a bot account and they are all by default marked <code>internal</code>. We used to be able to log in as the bot and change the user permissions and make the images public.</p>
<p>However, as of now, we are unable to make any newly uploaded image public. The <code>make public</code> button is greyed out for both the bot account (original uploader) and any other user who is given access to it (including the org owners).</p>
<p>We can do everything else like add more users, change their permissions, and connect to a repo. We just cannot make them public and they are stuck as <code>internal</code>.</p>
<p>Here are two images currently stuck:</p><aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/orgs/linuxserver/packages/container/package/hedgedoc" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/541623/181102120-250f17b6-e629-4554-9506-95ad1f0a1cf7.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/orgs/linuxserver/packages/container/package/hedgedoc" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/orgs/linuxserver/packages/container/package/hedgedoc" target="_blank" rel="noopener nofollow ugc">linuxserver/packages/container/package/hedgedoc</a></p>

  <p><span class="label1">GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<aside class="onebox githubfolder">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">
      <a href="https://github.com/orgs/linuxserver/packages/container/package/nntp2nntp" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:500/500;"><img src="https://user-images.githubusercontent.com/541623/181102121-3298ce9f-cf8d-43ee-91cb-69b0fe988f58.png" class="thumbnail" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png"></div>

<h3><a href="https://github.com/orgs/linuxserver/packages/container/package/nntp2nntp" target="_blank" rel="noopener nofollow ugc">Build software better, together</a></h3>

<p><a href="https://github.com/orgs/linuxserver/packages/container/package/nntp2nntp" target="_blank" rel="noopener nofollow ugc">linuxserver/packages/container/package/nntp2nntp</a></p>

  <p><span class="label1">GitHub is where people build software. More than 50 million people use GitHub to discover, fork, and contribute to over 100 million projects.</span></p>

  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>We’d appreciate you looking into it.</p>
<p>Thanks</p>

**Comment by aptalca** [comment]
<p><a class="mention" href="/clarkbw">@clarkbw</a> any ideas on who we can contact about this issue? It’s been impacting our users for weeks now as they are unable to pull these images (not to mention our github triggers are also failing due to not being able to check the last pushed version)</p>
<p>Thanks</p>

**Comment by shayneburgess** [comment]
<p><a class="mention" href="/aptalca">@aptalca</a> We are investigating this right now. Thanks for reporting this, i’ll reply when I know more about what is happening.</p>

**Comment by aptalca** [comment]
<p>Thanks for the confirmation :+1:</p>

**Comment by shayneburgess** [comment, marked as answer]
<p>No problem <a class="mention" href="/aptalca">@aptalca</a>. This should be fixed now and you should be able to make it public again. Please let me know if your still having issues.</p>

**Comment by aptalca** [comment]
<p>Excellent, it works again now. Thanks so much.</p>
